test/support/integration/enterprise_server.ex

```diff
@@ -0,0 +1,189 @@
+defmodule LivebookTest.EnterpriseServer do
```

Comment:
Should it be `Livebook.EnterpriseServer`?

Reply:
your call

Reply:
We don't use `LivebookTest` anywhere, so I would go with `Livebook.` for consistency yeah :)